src/data/tutorials.rs

```diff
@@ -50,9 +53,18 @@ impl DbKind for TutorialLesson {
         // TODO: register these as item Flags and check them for
         // the IsTutorialTagOpen gui function?
         // Downside: some mods may check other mods' tags for compatibility.
-        vd.field_value("gui_tag");
+        vd.multi_field_value("gui_tag");
         // TODO: check that this is a widget name
         vd.field_value("highlight_widget");
+        // TODO: verify this works in CK3 too
+        vd.field_validated_key("highlight_widget_dynamic_loc", |key, bv, data| {
+            validate_datatype_field(Datatype::Unknown, key, bv, data, false);
```

Comment:
should be `Datatype::CString`

Reply:
Unfortunately probably not. The game engine will convert numbers into strings too when it's expecting a string. Tiger doesn't currently have a way to express this. It's one of the TODOs in the gui code :)